picasso/src/main/java/com/squareup/picasso3/PicassoDrawable.java

```diff
@@ -53,6 +53,13 @@ static void setResult(ImageView target, Context context, RequestHandler.Result r
           new PicassoDrawable(context, bitmap, placeholder, loadedFrom, noFade, debugging);
       target.setImageDrawable(drawable);
     }
+    Drawable drawable = result.getDrawable();
+    if (drawable != null) {
```

Comment:
You either need a `hasDrawable()` method or to delete `hasBitmap()` on `Result`

Reply:
updated.